docs/audits.md

```diff
@@ -153,6 +153,9 @@ In practice, this means that workflows should almost always set
 `#!yaml permissions: {}` at the workflow level to disable all permissions
 by default, and then set specific job-level permissions as needed.
 
+Tools such as [GitHubSecurityLab/actions-permissions](https://github.com/GitHubSecurityLab/actions-permissions)
```

Comment:
Nitpick, nonblocking: if you wrap this in a `!!! tip` it might be a bit more prominent/easy to read. But I'll leave it at your discretion.

Reply:
Thanks for making these documentation changes!